MeetupPlatformApi/Controllers/MeetupsController.cs

```diff
@@ -1,98 +1,77 @@
 {
-    private readonly MeetupsPlatformContext _meetupsPlatformContext;
+    private readonly ApplicationContext context;
+    private readonly IMapper mapper;
 
-    private readonly IMapper _mapper;
-
-    public MeetupsController(
-        MeetupsPlatformContext meetupsPlatformContext,
-        IMapper mapper)
+    public MeetupsController(ApplicationContext context, IMapper mapper)
     {
-        _mapper = mapper;
-        _meetupsPlatformContext = meetupsPlatformContext;
+        this.context = context;
+        this.mapper = mapper;
     }
 
     [HttpGet]
     public async Task<IActionResult> GetAllMeetups()
     {
-        var meetupsEntity = await _meetupsPlatformContext.Meetups.ToListAsync();
-
-        var meetupsOutputDto = _mapper.Map<List<MeetupOutputDto>>(meetupsEntity);
-
-        return Ok(meetupsOutputDto);
+        var meetups = await context.Meetups.ToListAsync();
+        var outputDtos = mapper.Map<IEnumerable<MeetupOutputDto>>(meetups);
+        return Ok(outputDtos);
     }
 
-    [HttpGet("{id}")]
-    public async Task<IActionResult> GetMeetupById(int id)
+    [HttpGet("{id:int}")]
+    public async Task<IActionResult> GetMeetupById([FromRoute] int id)
     {
-        var meetupEntity = await _meetupsPlatformContext.Meetups.FirstOrDefaultAsync(meetup => meetup.Id == id);
-
-        var meetupOutputDto = _mapper.Map<MeetupOutputDto>(meetupEntity);
-
-        return meetupOutputDto is not null ? Ok(meetupOutputDto) : NotFound();
+        var meetup = await context.Meetups.SingleOrDefaultAsync(meetup => meetup.Id == id);
+        var outputDto = mapper.Map<MeetupOutputDto>(meetup);
+        return outputDto is not null ? Ok(outputDto) : NotFound();
     }
 
     [HttpPost]
-    public async Task<IActionResult> AddMeetup([FromBody] MeetupInputDto meetupInputDto)
+    public async Task<IActionResult> AddMeetup([FromBody] MeetupInputDto inputDto)
     {
-        var meetupEntity = _mapper.Map<MeetupEntity>(meetupInputDto);
+        var meetup = mapper.Map<MeetupEntity>(inputDto);
+        await context.Meetups.AddAsync(meetup);
+        await context.SaveChangesAsync();
 
-        await _meetupsPlatformContext.Meetups.AddAsync(meetupEntity);
-
-        await _meetupsPlatformContext.SaveChangesAsync();
-
-        var meetupOutputDto = _mapper.Map<MeetupOutputDto>(meetupEntity);
-
-        return CreatedAtAction(nameof(GetMeetupById), new { id = meetupOutputDto.Id }, meetupOutputDto);
+        var outputDto = mapper.Map<MeetupOutputDto>(meetup);
+        return CreatedAtAction(nameof(GetMeetupById), new { id = outputDto.Id }, outputDto);
     }
 
-    [HttpPut("{id}")]
-    public async Task<IActionResult> UpdateMeetup(int id, [FromBody] MeetupInputDto meetupInputDto)
+    [HttpPut("{id:int}")]
+    public async Task<IActionResult> UpdateMeetup([FromRoute] int id, [FromBody] MeetupInputDto inputDto)
     {
-        var exMeetupEntity = await _meetupsPlatformContext.Meetups
-            .AsNoTracking()
```

Comment:
I would strongly recommend against using `.AsNoTracking()` until you face performance issues. This options is a lot more dangerous and restricting than it looks on a first glance and it's hard to spot and fix issues, that can be caused by unconscious use of `.AsNoTracking()`.